server/app/socketService.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable consistent-return */
+const SocketIo = require('socket.io');
+const SocketClient = require('socket.io-client');
+const httpServer = require('http').createServer();
+const User = require('./models/User');
+
+const ENDPOINT = process.env.SOCKET_CHANNEL_ENDPOINT || 'http://localhost';
+const PORT = process.env.SOCKET_CHANNEL_PORT || 3002;
+
+const io = SocketIo();
```

Comment:
You can use the same server for both, actually. It would be smth like this:
```
const app = require("./app");
const http = require("http").Server(app);
const io = require("socket.io")(http);
...
const server = http.listen(
  PORT,
  console.log(
    `Server running in ${process.env.NODE_ENV} mode on port ${PORT}`
  )
);
```

Reply:
(on server.js)